coordinator/src/orderbook/websocket.rs

```diff
@@ -128,7 +128,7 @@ pub async fn websocket_connection(stream: WebSocket, state: Arc<AppState>) {
                     fcm_token,
                     signature,
                 }) => {
-                    let msg = create_sign_message();
+                    let msg = create_sign_message(b"Hello it's me Mario".to_vec());
```

Comment:
https://www.youtube.com/watch?v=BHPse3V1aMg

Reply:
Note, I just changed the signature to accept the message to be signed from the caller. The message `Hello it's me Mario` was there before (just moved to the caller).